.pipelines/templates/build-and-push-containers.yml

```diff
@@ -10,12 +10,15 @@ parameters:
 - name: IMAGE_PREFIX
   type: string
   default: ''
+- name: DOCKER_COMPOSE_FILE
+  type: string
+  default: 'docker-compose.yml'
```

Comment:
Thinking out load of options to avoid compose file duplication.
Optionally, compose receives service names so it can partially operate on part of the services. It could help here but I haven't found this option in the AZO task documentation page. 'Run' has 'serviceName' argument but no the others.
We can also use '[profiles](https://docs.docker.com/compose/profiles/)' and set COMPOSE_PROFILES env var to build part all / image / text. 
But this works as well...

Reply:
a possible V2.5 taks? :) lets brainstorm offline

Reply:
Sure,
If the 'beta' indication is more important than the major.minor versioning (somewhat makes sense since it uses v2 of the other packages), the we can also do:
1. 
```yml
  presidio-analyzer:
    image: ${REGISTRY_NAME}${IMAGE_PREFIX}presidio-analyzer${TAG}
...
  presidio-image-redactor:
    image: ${REGISTRY_NAME}${IMAGE_PREFIX}presidio-image-redactor${TAG}-beta
...

```
2. hardcode beta in setup.py

Reply:
we can just update the image-redactor version file